aspnetcore/includes/combine-di.md

```diff
@@ -0,0 +1,17 @@
+<a name="csc"></a>
+
+## Combining service collection
```

Comment:
It might be just me, but I read this as something that's going to cover how to take two `IServiceCollection` instances and combine them. I think it's a useful section, but I think it'd fit better inline in the DI topic, at the end of the _Register additional services with extension methods_ section. I also think that section could move down a bit, perhaps after _Service registration methods_.